pages/_app.js

```diff
@@ -40,19 +40,9 @@ function MyApp({ Component, pageProps, err  }) {
       `}
     </Script>
     <Header/>
-    <div className="govuk-width-container">
```

Comment:
Didn't notice this yesterday, but I see this div (lines 43 and 55) being removed.

I could be entirely wrong, but is there any chance that this div is what gives Hackney cantered layout with a bunch of empty space on the sides?

Perhaps, it would be worth double-checking?

Reply:
Addressed in PR #79